app/src/lib/title-bar-config.ts

```diff
@@ -0,0 +1,38 @@
+import { writeFile } from 'fs/promises'
+import { app } from 'electron'
+import { join } from 'path'
+import { TitleBarStyle } from '../ui/lib/title-bar-style'
+import { readFileSync } from 'fs'
+
+export type TitleBarConfig = {
+  titleBarStyle: TitleBarStyle
+}
+
+const getTitleBarConfigPath = () =>
+  join(app.getPath('userData'), '.title-bar-config')
+
+// The function has to be synchronous since we need its return value to create electron BrowserWindow
+function readTitleBarConfigFile(): TitleBarConfig {
+  let titleBarConfig: TitleBarConfig
+
+  try {
+    titleBarConfig = JSON.parse(readFileSync(getTitleBarConfigPath(), 'utf8'))
```

Comment:
:art: could we also skip this code if no file exists at this location?

Reply:
Wouldn't it be slower? (the function is synchronous)

But it is OK if you think that would be better.

Reply:
> Wouldn't it be slower? (the function is synchronous)

It's been a while since I've looked into the syscalls that are involved with filesystem access and errors that need to be handled but `stat` calls are probably less expensive (synchronous or otherwise).